svgelements/svgelements.py

```diff
@@ -111,6 +111,8 @@
 SVG_ATTR_RADIUS = "r"
 SVG_ATTR_POINTS = "points"
 SVG_ATTR_PRESERVEASPECTRATIO = "preserveAspectRatio"
+SVG_ATTR_START = "start"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute

Start is not an SVG_ATTR the attributes come in through `svg.path` and isn't part of the svg spec or part of the namespace.

Reply:
No - now I check the code they are used in PathSegment objects which are not part of the SVG spec. I will back these out too.